shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -30,22 +35,12 @@
  * - controller.hosts: metrics related to Pravega servers in the cluster (e.g., number of servers, failures)
  * - controller.container: metrics related to container lifecycle (e.g., failovers)
  *
- * We have two types of metrics:
```

Comment:
Why has this been deleted?

Reply:
It was mentioned in the past because metrics were named according to the different type. Metric names are now consistent regardless of the type.

Reply:
We are still using `_global` metrics, right? In fact, this is written in the paragraph below: `global version of it by adding "_global" suffix`. These two paragraphs are just elaborating on why we have metrics with an without `_global` suffix. Still, I don't see the need for removing this explanation, as it is linked with the paragraph below.

Reply:
Yes, added back.